README.md

```diff
@@ -11,12 +11,18 @@ and the Flutter guide for
 [developing packages and plugins](https://flutter.dev/developing-packages).
 -->
 
+<div align="center">
```

Comment:
Many other packages have badges just under the logo. I think it looks like aesthetically that logo is first and it naturally points user to the elements below icon

Reply:
I agree. Lets move it below logo.

Reply:
right, done